src/commands/init.ts

```diff
@@ -14,11 +14,21 @@ import {ArgDescriptor} from 'command-line-args';
 import * as fs from 'fs';
 import * as logging from 'plylog';
 import * as chalk from 'chalk';
-
+import * as YeomanEnvironment from 'yeoman-environment';
```

Comment:
we moved some imports into the `run()` method to improve start time of the CLI. Unless there's another reason to import up here, let's keep what we can in `run`.